packages/components/src/components/hds/dropdown/list-item/interactive.ts

```diff
@@ -25,15 +27,41 @@ export interface HdsDropdownListItemInteractiveSignature {
     text: string;
     trailingIcon?: FlightIconSignature['Args']['name'];
   };
+  Blocks: {
+    default?: [
+      {
+        Badge?: ComponentLike<HdsBadgeSignature>;
+      },
+    ];
+  };
   Element: HTMLDivElement | HdsInteractiveSignature['Element'];
 }
 
 export default class HdsDropdownListItemInteractiveComponent extends Component<HdsDropdownListItemInteractiveSignature> {
-  /**
-   * @param text
-   * @type {string}
-   * @description The text of the item. If no text value is defined an error will be thrown
-   */
+  constructor(
+    owner: unknown,
+    args: HdsDropdownListItemInteractiveSignature['Args']
+  ) {
+    super(owner, args);
+
+    if (args.text !== undefined) {
+      deprecate(
+        'The `@text` argument for "Hds::Dropdown::ListItem::Interactive" has been deprecated. Please put text in the yielded block.',
+        false,
+        {
+          id: 'hds.dropdown.list-item.interactive',
```

Comment:
I think this is lefover from the suggestion from copilt?

```suggestion
          id: 'hds.dropdown.list-item.text',
```

Reply:
Maybe it should be `hds.dropdown.list-item.interactive.text`

Reply:
oh yea. it could be. in practice it _really_ doesn't matter all that much so that seems good to me.